core/src/main/java/io/aiven/klaw/service/TopicOverviewService.java

```diff
@@ -290,6 +294,25 @@ private void updateTopicOverviewItems(
     }
   }
 
+  private boolean isDeleteRequestAlreadyOpen(
```

Comment:
looks like it is any request, instead of delete request ?

Reply:
If there is a request in that ENV we don't show the delete button.

Currently if there is a claim or another delete request open then we don't show the delete button.

Also if there is an Update we shouldn't IMO show an update. (although I am not sure if this is the current functionality)